packages/core/src/storage/doc-store/kv-document-store.ts

```diff
@@ -1,15 +1,15 @@
-import { DEFAULT_NAMESPACE } from "@llamaindex/core/global";
-import type { BaseNode } from "@llamaindex/core/schema";
-import { ObjectType } from "@llamaindex/core/schema";
-import type { RefDocInfo } from "@llamaindex/core/storage/doc-store";
+import clone from "lodash.clone";
+import get from "lodash.get";
```

Comment:
possible https://blog.saeloun.com/2023/03/02/replace-lodash-get-with-optional-chaining-and-nullish-coalescing-operator/ ?